pkg/gadgets/helpers.go

```diff
@@ -213,3 +214,31 @@ func FixBpfKtimeGetBootNs(programSpecs map[string]*ebpf.ProgramSpec) {
 		removeBpfKtimeGetBootNs(s)
 	}
 }
+
+func FilterMapReplacement(MountnsMap *ebpf.Map, spec *ebpf.CollectionSpec, objs interface{}) error {
```

Comment:
Some thoughts about this:
- What about making this function more generic, like `LoadeBPFSpec` since it's not only about replacing the filtering map?
- I think it can have a `consts map[string]interface{}` parameter so it can be used in other gadgets too. 
- It could also call `FixBpfKtimeGetBootNs` to avoid each gadget doing the same 
- https://github.com/inspektor-gadget/inspektor-gadget/pull/1504 could be implemented here to avoid code duplication

Reply:
``` 
> What about making this function more generic, like LoadeBPFSpec since it's not only about replacing the filtering map?
> I think it can have a consts map[string]interface{} parameter so it can be used in other gadgets too.
```
Added, Mauricio, could you please check?

```
> It could also call FixBpfKtimeGetBootNs to avoid each gadget doing the same
```
Good hint. I included `FixBpfKtimeGetBootNs` in the helper as well. 
I just have one mention, top/block-io gadget wasn't using `FixBpfKtimeGetBootNs`, but I think it makes no big difference that it's called now through `LoadeBPFSpec `.

CC: @mauriciovasquezbernal

Reply:
> I just have one mention, top/block-io gadget wasn't using FixBpfKtimeGetBootNs, but I think it makes no big difference that it's called now through LoadeBPFSpec .

Yes, I don't think it adds a huge overhead, so it's fine to call it for all gadgets.